### PR TITLE
Flashstake Strategy

### PR DESCRIPTION
**Pull request by anassohail99**
Fixes #. 

Changes proposed in this pull request:
- GitHub username
- get up to 1000 results
- query data for all the addresses.

### CODE REVIEW

File: src/strategies/flashstake/index.ts [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    FLASHSTAKE_SUBGRAPH_URL[network],
+    params
+  );
+
+  const stakesResult = await subgraphRequest(
+    FLASHSTAKE_SUBGRAPH_URL[network],
+    params2
+  );
+
+  console.log('OUTPUT', result, stakesResult);
```
Comment (ChaituVR) [inline src/strategies/flashstake/index.ts:80]: Hi @anassohail99 can we remove this log if possible, Thanks

**Review by ChaituVR — COMMENTED** [review]
LGTM @bonustrack

File: src/strategies/flashstake/index.ts (hunk 1 above)
Comment (anassohail99) [reply src/strategies/flashstake/index.ts:80]: Sure

File: src/strategies/flashstake/examples.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        "address": "0x20398ad62bb2d930646d45a6d4292baa0b860c1f",
+        "symbol": "FLASH",
+        "decimals": 18
+      }
+    },
+    "network": "1",
+    "addresses": [
+      "0x3A1776aD8Bd41c688019Cb586C6A1261Be49f8Bf"
+    ],
+    "snapshot": "latest"
```
Comment (bonustrack) [inline src/strategies/flashstake/examples.json:16]: This should use a specific block number
```suggestion
    "snapshot": 12359920
```

**Pull request merged**